benchmarks/tpcds/schemas/s64da_columnstore/schema.sql

```diff
@@ -0,0 +1,1012 @@
+    ws_net_paid_inc_tax       DOUBLE PRECISION,
+    ws_net_paid_inc_ship      DOUBLE PRECISION,
+    ws_net_paid_inc_ship_tax  DOUBLE PRECISION,
+    ws_net_profit             DOUBLE PRECISION
+);
+
+CREATE TABLE catalog_sales
+(
+    cs_sold_date_sk           INTEGER,
+    cs_sold_time_sk           INTEGER,
+    cs_ship_date_sk           INTEGER,
+    cs_bill_customer_sk       INTEGER,
+    cs_bill_cdemo_sk          INTEGER,
+    cs_bill_hdemo_sk          INTEGER,
+    cs_bill_addr_sk           INTEGER,
+    cs_ship_customer_sk       INTEGER,
+    cs_ship_cdemo_sk          INTEGER,
+    cs_ship_hdemo_sk          INTEGER,
+    cs_ship_addr_sk           INTEGER,
+    cs_call_center_sk         INTEGER,
+    cs_catalog_page_sk        INTEGER,
+    cs_ship_mode_sk           INTEGER,
+    cs_warehouse_sk           INTEGER,
+    cs_item_sk                INTEGER               NOT NULL,
+    cs_promo_sk               INTEGER,
+    cs_order_number           INTEGER               NOT NULL,
+    cs_quantity               BIGINT,
+    cs_wholesale_cost         DOUBLE PRECISION,
+    cs_list_price             DOUBLE PRECISION,
+    cs_sales_price            DOUBLE PRECISION,
+    cs_ext_discount_amt       DOUBLE PRECISION,
+    cs_ext_sales_price        DOUBLE PRECISION,
+    cs_ext_wholesale_cost     DOUBLE PRECISION,
+    cs_ext_list_price         DOUBLE PRECISION,
+    cs_ext_tax                DOUBLE PRECISION,
+    cs_coupon_amt             DOUBLE PRECISION,
+    cs_ext_ship_cost          DOUBLE PRECISION,
+    cs_net_paid               DOUBLE PRECISION,
+    cs_net_paid_inc_tax       DOUBLE PRECISION,
+    cs_net_paid_inc_ship      DOUBLE PRECISION,
+    cs_net_paid_inc_ship_tax  DOUBLE PRECISION,
+    cs_net_profit             DOUBLE PRECISION
+);
+
+CREATE TABLE store_sales
+(
+    ss_sold_date_sk           INTEGER,
+    ss_sold_time_sk           INTEGER,
+    ss_item_sk                INTEGER               NOT NULL,
+    ss_customer_sk            INTEGER,
+    ss_cdemo_sk               INTEGER,
+    ss_hdemo_sk               INTEGER,
+    ss_addr_sk                INTEGER,
+    ss_store_sk               INTEGER,
+    ss_promo_sk               INTEGER,
+    ss_ticket_number          INTEGER               NOT NULL,
+    ss_quantity               BIGINT,
+    ss_wholesale_cost         DOUBLE PRECISION,
+    ss_list_price             DOUBLE PRECISION,
+    ss_sales_price            DOUBLE PRECISION,
+    ss_ext_discount_amt       DOUBLE PRECISION,
+    ss_ext_sales_price        DOUBLE PRECISION,
+    ss_ext_wholesale_cost     DOUBLE PRECISION,
+    ss_ext_list_price         DOUBLE PRECISION,
+    ss_ext_tax                DOUBLE PRECISION,
+    ss_coupon_amt             DOUBLE PRECISION,
+    ss_net_paid               DOUBLE PRECISION,
+    ss_net_paid_inc_tax       DOUBLE PRECISION,
+    ss_net_profit             DOUBLE PRECISION
+);
+
+CREATE INDEX customer_address_cache ON customer_address USING columnstore (
```

Comment:
Are all these columns valid for indexes according to TPC-DS specs? If not, are ignoring compliance for now?

Reply:
i have no idea. however its not a traditional index which allows pointlookups so i would go for "ignore compliance" for now

Reply:
from a quick glance in http://www.tpc.org/tpc_documents_current_versions/pdf/tpc-h_v2.17.1.pdf on 1.5.6 and 1.5.7 it looks okay-ish?

Reply:
The link is to tpc-h. In this specific case it's section 2.5.3 in http://www.tpc.org/tpc_documents_current_versions/pdf/tpc-ds_v2.13.0.pdf

But as discussed elsewhere, we'll ignore that for now, and have another task to verify the indexes.